configure.ac

```diff
@@ -1151,6 +1151,9 @@ fi
 
 if test "$enable_afsnmp" = "yes"; then
   NETSNMP_CFLAGS="`$NETSNMP_CONFIG --cflags`"
+  # net-snmp-config --cflags could return bogus, invalid paths on arm/macOS Ventura, dirty, quick cleanup till net-snmp-config is not fixed
+  # take care, autotools require double [[]] here!!!
+  NETSNMP_CFLAGS=${NETSNMP_CFLAGS/[[ \t]]-[[0-9]]/}
```

Comment:
I think it's not worth adding such a nasty workaround to our configure script, because one can install net-snmp from brew as a workaround (only the system-provided net-snmp is bogus).

Reply:
indead, will revert back this and add the environment modification requiremnts for net-snmp to the macos build documentation